deploy/docker/README.md

```diff
@@ -20,7 +20,17 @@ $ docker run -d \
     --net=host \
     --pid=host \
     --privileged \
-    gcr.io/google_containers/localkube-amd64:vX.Y.Z \
+    gcr.io/google_containers/localkube-amd64:DESIRED_TAG \
```

Comment:
we should probably change this since we don't actually push to this repo. we have k8s-minikube, want to push there?